Lib/subprocess.py

```diff
@@ -815,6 +839,13 @@ def __init__(self, args, bufsize=-1, executable=None,
         self.encoding = encoding
         self.errors = errors
         self.pipesize = pipesize
+        if read_callback is not None:
+            if not callable(read_callback):
+                ValueError("read_callback not a callable")
```

Comment:
missing raise and associated test case thus ;)

Reply:
> missing raise and associated test case thus ;)

Done